packages/dev/addons/src/lottie/worker.ts

```diff
@@ -0,0 +1,47 @@
+import type { Nullable } from "core/types";
+
+import type { AnimationConfiguration } from "./lottiePlayer";
+import { AnimationController } from "./rendering/animationController";
+
+let AnimationPlayer: Nullable<AnimationController> = null;
+
+/**
+ * Default configuration for lottie animations playback.
+ */
+const DefaultConfiguration: AnimationConfiguration = {
+    loopAnimation: false, // By default do not loop animations
+    spriteAtlasSize: 2048, // Size of the texture atlas
+    gapSize: 5, // Gap around the sprites in the atlas
+    spritesCapacity: 64, // Maximum number of sprites the renderer can handle at once

```

Comment:
Is this necessary ? can it not be inferred from the loaded file ?

Reply:
Is it a lot harder (only for the renderer creation) to be deferred ? This will change the public API later to remove the spritesCapacity which should always be as tight as possible for memory/perf purposes

Reply:
Would be great to address soon if you can make a subsequent PR ?